### PR TITLE
docs(scalasdk): Scala quickstart customer registry

### PR DESCRIPTION
**Pull request by johanandren**

### CODE REVIEW

File: samples/scala-customer-registry-quickstart/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# Quickstart project: Customer Registry
+
+# TODO update for quickstart instructions
```
Comment (patriknw) [inline samples/scala-customer-registry-quickstart/README.md:3]: still something TODO?

File: samples/scala-customer-registry-quickstart/build.sbt [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Test / parallelExecution := false
+Test / testOptions += Tests.Argument("-oDF")
+Test / logBuffered := false
+
+Compile / run := {
+  // needed for the proxy to access the user function on all platforms
+  sys.props += "akkaserverless.user-function-interface" -> "0.0.0.0"
+  (Compile / run).evaluated
+}
```
Comment (patriknw) [inline samples/scala-customer-registry-quickstart/build.sbt:33]: @raboof would this be something the sbt-plugin could set, to simplify the user's build?

**Review by patriknw — APPROVED** [review]
LGTM, will you create the .adoc in the -docs repo also?

File: samples/scala-customer-registry-quickstart/README.md (hunk 1 above)
Comment (johanandren) [reply samples/scala-customer-registry-quickstart/README.md:3]: Copied TODO from the Java README, not quite sure what it is about, but if it is needed there it probably is needed here?

**Comment by johanandren** [comment]
Yes, started on a doc page in the docs repo.

**Pull request merged**